python/python/glide/async_commands/server_modules/json.py

```diff
@@ -1,18 +1,18 @@
 # Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
-"""module for `RedisJSON` commands.
+"""module for `JSON` commands.
```

Comment:
consider renaming to glide_json.py
```suggestion
"""Glide module for `JSON` commands.
```

Reply:
I will